cpython-unix/extension-modules.yml

```diff
@@ -806,6 +856,7 @@ readline:
     - readline.c
   defines:
     - USE_LIBEDIT=1
+    - Py_RL_STARTUP_HOOK_TAKES_ARGS
```

Comment:
Can we document this?

Reply:
cc @kpfleming no clue what this is for

Reply:
It's from here: https://github.com/python/cpython/blob/d343f977ba89c415aa4dcaf75122e0b4235d4fb1/configure.ac#L6517

Reply:
And used here: https://github.com/python/cpython/blob/d343f977ba89c415aa4dcaf75122e0b4235d4fb1/Modules/readline.c#L1044

Reply:
And we need to set it because... our editline version takes arguments?

Reply:
I think I get this now. Added a comment.